### PR TITLE
Feature/not found page

### PR DESCRIPTION
**Pull request by L1ght-time**

### CODE REVIEW

File: src/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  useEffect(() => {
+    setTimeout(() => {
+      return router.push(ROUTES_PAGES.home);
+    }, 5000);
+  }, [router]);
+
+  return (
+    <div className="flex justify-center items-center h-full">
+      <div>
+        <button onClick={() => router.push(ROUTES_PAGES.home)}>
```
Comment (mykyta-lazurenko) [inline src/pages/404.tsx:20]: Create handeRedirect and reuse in useEffect

File: src/pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { TbArrowNarrowLeft } from 'react-icons/tb';
+import { FormattedMessage } from 'react-intl';
+
+import { ROUTES_PAGES } from '@/constants';
+
+const NotFound = () => {
+  const router = useRouter();
+
+  useEffect(() => {
+    setTimeout(() => {
```
Comment (mykyta-lazurenko) [inline src/pages/404.tsx:12]: Maybe better useTimeoutFn?

Comment (L1ght-time) [reply src/pages/404.tsx:12]: Not for this case, because we can't call hooks inside useEffect.

Comment (mykyta-lazurenko) [reply src/pages/404.tsx]: Use without `useEffect`, what problem? Did you see how works `useTimeoutFn` hook?

**Review by mykyta-lazurenko — COMMENTED** [review]
See more details how works `useTimeoutFn` hook

File: src/pages/404.tsx (hunk 2 above)
Comment (L1ght-time) [reply src/pages/404.tsx]: Yes, but I tried to call hook inside useEffect.

**Review by mykyta-lazurenko — APPROVED** [review]

**Pull request merged**